src/helix/core/HelixSprite.hx

```diff
@@ -142,6 +147,35 @@ class HelixSprite extends FlxSprite
         }
         return y;
     }
+    
+    private function resetAcceleration():Void

```

Comment:
I don't see value in this function being here -- it's too specific to our current game (Helix is supposed to be generic). Can we move this back into the ship class? Can you also use parameters to `set`, so that readers don't need to look up the default values to figure out what it's doing?